e2e/cypress/support/commands/form.js

```diff
@@ -2,6 +2,11 @@ import { confirmCalendarDay } from '../functions';
 import { RewriteURL } from '../utils/constants';
 
 // thx to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace
+/**
+ * TODO tbp: I don't know what's the point of this function.
+ *    And since idk what it does, I want to remove it as it looks useless to me.
+ *    Check what this does and remove it if it's useless
+ */
```

Comment:
I see it is used in the form several times used in matching the patch value.  So not sure if it's a good idea to remove it